Mathlib/Analysis/Normed/Algebra/Spectrum.lean

```diff
@@ -65,26 +65,45 @@ section SpectrumCompact
 
 open Filter
 
-variable [NormedField 𝕜] [NormedRing A] [NormedAlgebra 𝕜 A]
+variable [NormedField 𝕜]
```

Comment:
You won't be able to get much out of `spectralRadius` if your algebra isn't a `NormedAlgebra`, but I guess you probably have matrices in mind. Even there, your best bet will be to activate a local instance. Nevertheless, we can leave this. But instead of adding the hypotheses to each declaration, just create a small section.